internetarchive/openlibrary
# Fix graphs not rendering

Closes #3940 



### Technical

Graphs were not rendering due to at least one variable in flot 0.7 not being declared before use.  This PR upgrades Flot to 0.8.3, which has the same API while properly declaring variables.

The next major version available via npm is 2.1.2.  It will require a code change in order to use.

### Testing

1. Install new packages
2. Build JavaScript assets
3. Navigate to https://openlibrary.org and ensure that graphs are rendering properly at the bottom of the page.
4. Ensure that graphs are rendering at https://openlibrary.org/stats
### Screenshot

![flot_0_8_0-alpha](https://user-images.githubusercontent.com/28732543/98069491-e5ef6d80-1e2c-11eb-8956-fe12ae32fd62.png)

**Note:** Disregard the 0's underneath each of the graphs.  I hard coded production data into my local `stats.html` in order to get the screenshot of the graphs.
### Stakeholders
